server/schedule/operator/operator.go

```diff
@@ -363,3 +367,11 @@ func (o *Operator) GetAdditionalInfo() string {
 	}
 	return ""
 }
+
+// mock region default region size is 96Mb.
+const mockRegionSize = 96 * (1 << 20)
+
+// NewTestOperator creates a test operator.
+func NewTestOperator(desc, brief string, regionID uint64, regionEpoch *metapb.RegionEpoch, kind OpKind, steps ...OpStep) *Operator {
```

Comment:
Why do we need it?

Reply:
This constructor of the operator is used for unit test, it will decrease works for unit test.